### PR TITLE
Make sign in label visiable when criteria met

### PR DESCRIPTION
**Pull request by Rambowalsh**
### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/GP2-2044
- [x] Jira ticket has the correct status.
- [x] [Changelog](CHANGELOG.md) entry added.

### Merging

- [x] This PR can be merged by reviewers. (If unticked, please leave for the author to merge)

### CODE REVIEW

File: tests/unit/domestic/test_models.py [hunk 1]
```diff
@@ -1685,6 +1685,7 @@ def test_get_context(self, mock_get_sector_list):
         )
 
 
+# @pytest.mark.skip(reason='no way of currently testing this')
```
Comment (bobmeredith) [inline tests/unit/domestic/test_models.py:1688]: Is this meant to be here?
There don't seem to be any tests for this flag. Should be quite easy to add unit tests on landing page, logged in and out yes?

**Review by bobmeredith — COMMENTED** [review]
Looks OK. You should probably have some tests though.

File: tests/unit/domestic/test_models.py (hunk 1 above)
Comment (Rambowalsh) [reply tests/unit/domestic/test_models.py:1688]: Steves original test has been fixed now so i'll remove the commented skip

File: CHANGELOG.md [hunk 2]
```diff
@@ -46,7 +46,8 @@ GP2-1849 - Feature flag magna links in logged out mobile header
 - GP2-1630 - add more data
 
 ### Implemented enhancements
-
+- GP2-2044 - amended to only show on homepage
```
Comment (stevejalim) [inline CHANGELOG.md:49]:
```suggestion
```
We can just drop this line I think as it's part of the original story

**Review by stevejalim — APPROVED** [review]
Seems OK to me - would be good to have tests, but this gets close manual QA anyway. Am assuming you've run it locally already

File: CHANGELOG.md (hunk 2 above)
Comment (Rambowalsh) [reply CHANGELOG.md]: Ok Removed

**Pull request merged**